src/gallia/services/uds/server.py

```diff
@@ -871,6 +871,7 @@ async def handle_client(
                     await writer.drain()
             except Exception:
                 traceback.print_exc()
+                break
```

Comment:
As far as I understood @fkglr prefers to specify certain exceptions that break the loop while others are ignored?

Reply:
`TCPUDSServerTransport`  accepts new connections anyhow, so we fail for all errors.